blockchain/chain_sync/src/chain_muxer.rs

```diff
@@ -176,7 +176,7 @@ where
         tipset_sender: Sender<Arc<Tipset>>,
         tipset_receiver: Receiver<Arc<Tipset>>,
         cfg: SyncConfig,
-    ) -> Result<Self, ChainMuxerError<C>> {
+    ) -> Result<Self, Box<ChainMuxerError<C>>> {
```

Comment:
This is still required to box `ChainMuxerError` now that you've boxed `TipsetValidationError`?

Reply:
Removed